app/assets/images/icons/arrow-top-right-on-square.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-6">
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor">
```

Can you remove this class? We don't have/want/need it, but i want to avoid any weird issues if we ever add this